### PR TITLE
order book queries for levels 1, 2 and 3

### PR DESCRIPTION
**Pull request by mnaamani**
- Updated database schema and relevant files d11ca529513bbf4f79e75c822e5694084e747b60
- OrderBook queries (read only)
- Reverting back to use process.env.NODE_ENV to configure knex, it is much cleaner approach.
The default environment is `development` so run knex migrations and seeding on test database (flyptox_test)

`knex --env test migrate:latest`
`knex --env test seed:run`

To run the server unit tests on the test db:

`NODE_ENV=test grunt test:server`

### CODE REVIEW

File: db/migrations/20150714121034_add_trades.js [hunk 1]
```diff
@@ -3,11 +3,11 @@
 exports.up = function(knex, Promise) {
   return Promise.all([
     knex.schema.createTable('trades', function(table) {
-      table.uuid('id').primary().notNullable();
```
Comment (recursion) [inline db/migrations/20150714121034_add_trades.js]: Im curious about removing notNullable() here? What is the reasoning behind this? 
It seems to me that we would never want a trade to be created without an ID?

**Comment by RWTreichel** [comment]
,primary() implies .notNullable(), so it should still be there in spirit I think.

File: server/controllers/book.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
+var OrderBook = module.exports;
```
Comment (recursion) [inline server/controllers/book.js]: So I agree with what you said about just getting this working for now and working on the more sophisticated/speedy approach later - but we definitely need to take a look at designing a system that runs the book in memory. We can keep a copy in the db, that sort of gets async writes when book updates happen - but in general order matching, and book data should happen directly in memory, yes?

**Comment by recursion** [comment]
Ok! Awesome! I was wondering about primary()'s implications on notNullable(). Great. So anywhere you declare primary, notNullable is assumed.

**Pull request merged**